OracleUnifiedDirectorySM/dockerfiles/12.2.1.4.0/README.md

```diff
@@ -0,0 +1,116 @@
+
+This image refers to binaries for OUD Release 12.2.1.4.0 and it has the capability to create an FMW Infrastructure domain with the OUDSM application deployed in a container with OUDSM deployed which can be targeted for development and testing.
+
+***Image***: `oracle/oudsm:12.2.1.4.0`
+
+# 2. Hardware and Software Requirements
+Oracle Unified Directory Service Managed Services image has been tested and is known to run on following hardware and software:
+
+## 2.1 Hardware Requirements
+
+| Hardware  | Size  |
+| :-------: | :---: |
+| RAM       | 16GB  |
+| Disk Space| 200GB+|
+
+## 2.2 Software Requirements
+
+|       | Version                        | Command to verify version |
+| :---: | :----------------------------: | :-----------------------: |
+| OS    | Oracle Linux 7.3 or higher     | more /etc/oracle-release  |
+| Docker| Docker version 18.03 or higher | docker version            |
+
+# 3. Build Oracle JDK/Oracle FMW Infrastructure image
+
+To build an OUDSM image either you can start from building the Oracle JDK and Oracle Fusion Middleware Infrastrucure image or use the already available Oracle JDK/Oracle Fusion Middleware Infrastructure image. The images are available in the [Oracle Container Registry](https://container-registry.oracle.com), and can be pulled from there. Steps to obtain the images from the [Oracle Container Registry](https://container-registry.oracle.com), or to build the images are provided in the steps below:
+
+## 3.1 Pulling the Oracle JDK (Server JRE) base image
+You can pull the Oracle Server JRE 8 image from the [Oracle Container Registry](https://container-registry.oracle.com). When pulling the Server JRE 8 image, re-tag the image so that it works with the dependent Dockerfiles which refer to the JRE 8 image through oracle/serverjre:8.
+
+**IMPORTANT**: Before you pull the image from the registry, please make sure to log-in through your browser with your SSO credentials and ACCEPT "Terms and Restrictions".
+
+1. Sign in to [Oracle Container Registry](https://container-registry.oracle.com). Click the **Sign in** link which is on the top-right of the Web page.
+2. Click **Java** and then click on **serverjre**.
+3. Click **Accept** to accept the license agreement.
+4. Use following commands to pull Oracle Fusion Middleware infrastructure base image from repository :
+
+        
+        $ docker login container-registry.oracle.com
+        $ docker pull container-registry.oracle.com/java/serverjre:8
+        $ docker tag container-registry.oracle.com/java/serverjre:8 oracle/serverjre:8
+
+## 3.2 How to build the Oracle Java image
+
+Please refer to [README.md](https://github.com/oracle/docker-images/blob/master/OracleJava/README.md) under docker/OracleJava for details on how to build Oracle Database image.
+
+https://github.com/oracle/docker-images/tree/master/OracleJava/README.md
+
+## 3.3 Pulling Oracle FMW Infrastructure 12.2.1.4.x image
```

Comment:
This is duplicated from above and isn't required either. Remove this section from the repo docs. We only document how to build, not how to pull.